Makefile

```diff
@@ -105,7 +106,14 @@ test.ci: generate
 .PHONY: e2e
 e2e: $(call GETENVOY_OUT_PATH,$(GOOS),$(GOARCH))
 	docker-compose up -d
-	go test $(E2E_OPTS) $(E2E_EXTRA_OPTS) $(E2E_PKG_LIST)
+	$(RUN_E2E)
+
+# Runs e2e for all extensions languages in pkg/extension/workspace/config/extension/helpers.go
+.PHONY: e2e.all
+e2e.all: $(call GETENVOY_OUT_PATH,$(GOOS),$(GOARCH))
+	docker-compose up -d
+	E2E_EXTENSION_LANGUAGE=rust $(RUN_E2E)
```

Comment:
note: this is intentionally not fancy, as calling something twice is easier to understand than for loops in make